backend/inspirehep/mailing/providers/sendgrid.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import structlog
+from flask import current_app
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+from inspirehep.mailing.errors import SendGridMissingAPIToken
+
+LOGGER = structlog.getLogger()
+
+
+def send_email(sender, recipient, subject, content, cc=()):
```

Comment:
IMHO its just more readable and less confusing to set cc to None and do ```if cc:``` check in the code.